examples/translation-job.json

```diff
@@ -0,0 +1,17 @@
+{
+    "id": "cf5e1c7c-86ae-4f03-aa9e-513b46cd3b7e",
+    "created_on": "2016-10-11T12:11:32+02:00",
```

Comment:
2013-03-01T16:15:09+01:00
2013-03-01T16:15:09+0100

Both of these strings are valid ISO8601 datetime strings, but the latter is not accepted by the constructor of JavaScript's date object in many browsers

Plus is also the recommendation in the PHP guide 

![2016-11-29 18_03_12-php_ datetime - manual](https://cloud.githubusercontent.com/assets/7221909/20720059/2c86d87c-b65e-11e6-96ad-da61147a72e7.png)